manubot/process/bibliography.py

```diff
@@ -63,9 +62,9 @@ def load_manual_references(paths=[], extra_csl_items=[]) -> dict:
         bibliography = load_bibliography(path)
         for csl_item in bibliography:
             csl_item.note_append_text(
-                f"This CSL JSON Item was loaded by Manubot v{manubot_version} from a manual reference file."
+                f"Loaded from an external bibliography file by Manubot."
```

Comment:
The version number here was making testing difficult by changing the expected output upon every release.